### ISSUE TITLE
Ignore style tags when type attribute is not "text/css"

### ISSUE THREAD

**Issue by gmetais** [macbre/phantomas#694]
Currently, when run with `--analyze-css`, Phantomas tries to parse the content of every `<style>` tags, whatever the **type** attribute is.

Phantomas should ignore any type that's not `text/css` (which is the default type), such as:
```
<style type="junk"> do not complain about this. </style>
```
and don't mark them as `cssParsingErrors`.

----

Originally reported by @a701440 / https://github.com/gmetais/YellowLabTools/issues/229

**Comment by macbre** [comment]
```
15:19:02.838 analyzeCss: inline <style> tag found with type="junk", ignoring...
```

will be logged in such cases

**Comment by macbre** [comment]
Fixed in #700, thanks for the bug report, @gmetais !
